layers/core_checks/cc_shader_interface.cpp

```diff
@@ -547,9 +547,20 @@ bool CoreChecks::ValidateFsOutputsAgainstRenderPass(const spirv::Module &module_
     return skip;
 }
 
+static std::string DescribeMappedLocation(uint32_t shader, uint32_t rendering_info) {
+    std::stringstream msg;
+    if (shader == rendering_info) {
+        msg << shader;
+    } else {
+        msg << shader << ", which was remapped to attachment " << rendering_info << ",";
```

Comment:
```suggestion
        msg << shader << " (which was remapped to attachment " << rendering_info << ") ";
```

also works, will leave it up to you to decide

Reply:
Went with this.